pyEpiabm/pyEpiabm/tests/test_unit/test_intervention/test_travel_isolation.py

```diff
@@ -77,13 +79,15 @@ def test___call__(self, mock_random):
         self.assertIsNone(self.person_introduced.travel_isolation_start_time)
         self.assertEqual(len(self._microcell.households), 1)
         self.assertEqual(len(self.person_symp.household.persons), 3)
-
+        mock_log.assert_called()
```

Comment:
I really don't like this function. Can we check it was called the right number of times (and ideally with the right argument)

Reply:
Will do